### ISSUE TITLE
Donation Preview Image for Facebook

### ISSUE THREAD

**Issue by MelissaAutumn** [thunderbird/thunderbird-website#518]
Currently sharing on Facebook produces the small square Thunderbird icon as the preview image, which works for the majority of cases. However after chatting with @KillYourFM it became apparent that a custom tailored preview image for the donation page could be valuable in terms of user engagement. 

We should look into adding a large-form og:image for the donation page. 

Preview images can be debugged via https://developers.facebook.com/tools/debug/?q=https%3A%2F%2Fwww.thunderbird.net (which sadly requires you to be logged in..)

**Comment by Sancus** [comment]
![image](https://github.com/thunderbird/thunderbird-website/assets/479671/087778e0-e8d0-4f5c-ae6d-5b5dc17a8661)

You mean this?

**Comment by MelissaAutumn** [comment]
Yep but a separate 1200x630px version for just the donation page.
